src/data/resolvers/stakes.ts

```diff
@@ -0,0 +1,57 @@
+import { Resolvers } from '@apollo/client';
+import { ClientType, Extension } from '@colony/colony-js';
+import { bigNumberify } from 'ethers/utils';
+
+import { Context } from '~context/index';
+
+export const stakesResolvers = ({
+  colonyManager,
+}: Required<Context>): Resolvers => ({
+  Query: {
+    async stakeAmountsForMotion(
+      _,
+      { colonyAddress, userAddress, motionId, isObjectionStake, tokenDecimals },
+    ) {
+      try {
+        const supportedSide = isObjectionStake ? 0 : 1;
```

Comment:
These need to be types. I think we already have them somewhere in the dapp, and I'm also going to add them to `colonyJS

Reply:
Didn't found it on the dapp, but I'm gonna add it to `~utils/colonyMotions`